config-templates/redhat/odf-remote/4.14/custom-parameters.json

```diff
@@ -258,6 +258,6 @@
     "supported-values": ["true", "false"],
     "type": "boolean",
     "required": "true",
-    "mutable": "true"
+    "mutable": "false"
```

Comment:
I think this should be `mutable` 
it should be "mutable": "true"

Reply:
This parameter cannot be updated after installation